packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -3842,11 +3849,14 @@ namespace Ifpack2 {
 
               // y := x - R y
               Z.doImport(Y, *tpetra_importer, Tpetra::REPLACE);
+              const auto YY = Y.getLocalViewDevice(Tpetra::Access::ReadWrite);
+              const auto ZZ = Z.getLocalViewDevice(Tpetra::Access::ReadWrite);
```

Comment:
Here as well. If we need to this way to remove host/device alive error, it means that we use a host view somewhere in this code or tpetra code. If possible, we need to remove that case.